### PR TITLE
Make all URLs lowercase

### PR DESCRIPTION
**Pull request by Clarity-89**
Ensure all the website URLs are lowercase, for SEO purposes. 

The solution is to rename all the files and directories to be lowercase and use [category item metadata](https://docusaurus.io/docs/sidebar/autogenerated#category-item-metadata) to make category items uppercase in sidebar.

### CODE REVIEW

File: docs/03-foundations/_category_.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "label": "Foundations",
+  "collapsible": true,
+  "position": 1,
```
Comment (ashharrison90) [inline docs/03-foundations/_category_.json]: do we actually want `position: 1` here?

File: docs/01-about/_category_.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "label": "About",
+  "collapsible": true,
+  "collapsed": true
```
Comment (ashharrison90) [inline docs/01-about/_category_.json:4]: question for curiosity - what happens if one/both of these aren't present? is the category permanently expanded by default? 🤔

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍

File: docs/03-foundations/_category_.json (hunk 1 above)
Comment (Clarity-89) [reply docs/03-foundations/_category_.json]: Yeah, no need, removed it.

File: docs/01-about/_category_.json (hunk 2 above)
Comment (Clarity-89) [reply docs/01-about/_category_.json:4]: Looks like both are true by default, so I'll remove them.

**Pull request merged**